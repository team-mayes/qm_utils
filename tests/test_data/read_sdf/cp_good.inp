14b.sdf  0.767  -1.253   0.241  1.279   0.000  -0.482  0.767   1.253   0.241 -0.767   1.253   0.241 -1.279   0.000  -0.482 -0.767  -1.253   0.241 
