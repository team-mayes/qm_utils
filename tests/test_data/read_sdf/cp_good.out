"name","phi","theta","q","closest_pucker"
"14b.sdf",240.00000000000003,90.0,0.8348484892481989,"14b"
"1c4.sdf",83.6598082540901,180.0,0.6270693741524935,"1c4"
"3e.sdf",0.5165961211938987,125.16673486810083,0.4766146774356231,"3e"
